### PR TITLE
New terminology to be closer to IOTA 2.0

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/CHANGELOG.md [hunk 1]
```diff
@@ -166,7 +166,7 @@
 - Backup destination can now be a path to a file instead of a directory, allowing custom filenames.
   - [ffbeaa3](https://github.com/iotaledger/wallet.rs/commit/ffbeaa3466b44f79dd5f87e14ed1bdc4846d9e85) feat(backup): allow file path as destination for custom filenames ([#426](https://github.com/iotaledger/wallet.rs/pull/426)) on 2021-03-14
 - Adds a `messageId` field to the balance change event payload.
-  - [c7d34e2](https://github.com/iotaledger/wallet.rs/commit/c7d34e213a7a42503b21714847c9642e19878cb4) feat(events): reintroduce message_ids on balance change event payload ([#406](https://github.com/iotaledger/wallet.rs/pull/406)) on 2021-03-09
+  - [c7d34e2](https://github.com/iotaledger/wallet.rs/commit/c7d34e213a7a42503b21714847c9642e19878cb4) feat(events): reintroduce block_ids on balance change event payload ([#406](https://github.com/iotaledger/wallet.rs/pull/406)) on 2021-03-09
```
Comment (Thoralf-M) [inline bindings/nodejs/CHANGELOG.md]: Changelog shouldn't be modified

File: documentation/docs/specification.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ keywords:
...
+The package introduces the concept of an _account_ . An _account_ is a reference to, or a label for, a [seed](https://chrysalis.docs.iota.org/guides/dev_guide#seed). An account has certain properties such as [addresses](https://github.com/Wollac/protocol-rfcs/blob/bech32-address-format/text/0020-bech32-address-format/0020-bech32-address-format.md) and [blocks](https://github.com/GalRogozinski/protocol-rfcs/blob/block/text/0017-block/0017-block.md). An account has various possible behaviours, including moving funds, looking for new blocks, and making copies of block histories. An account should also be able to provide a degree of financial privacy and this should not incur any overhead.
```
Comment (Thoralf-M) [inline documentation/docs/specification.md]: Link doesn't exist

File: src/account/operations/output_collection.rs [hunk 3]
```diff
@@ -368,7 +368,7 @@ impl AccountHandle {
                 Ok(res) => {
                     log::debug!(
                         "[OUTPUT_COLLECTION] Collection transaction created: msg_id: {:?} tx_id: {:?}",
```
Comment (Thoralf-M) [inline src/account/operations/output_collection.rs]: There is still `msg_id`

File: src/account/operations/output_consolidation.rs [hunk 4]
```diff
@@ -95,7 +95,7 @@ impl AccountHandle {
                     Ok(res) => {
                         log::debug!(
                             "[OUTPUT_CONSOLIDATION] Consolidation transaction created: msg_id: {:?} tx_id: {:?}",
```
Comment (Thoralf-M) [inline src/account/operations/output_consolidation.rs]: There is still msg_id

File: src/account/operations/syncing/transactions.rs [hunk 5]
```diff
@@ -141,7 +141,7 @@ impl AccountHandle {
         for mut transaction in transactions_to_reattach {
             log::debug!("[SYNC] reattach transaction");
             let reattached_msg = self.submit_transaction_payload(transaction.payload.clone()).await?;
-            transaction.message_id.replace(reattached_msg);
+            transaction.block_id.replace(reattached_msg);
```
Comment (Thoralf-M) [inline src/account/operations/syncing/transactions.rs]: Should be `reattached_block`

**Review by Thoralf-M — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**